### DISCUSSION TITLE
绳网

### DISCUSSION THREAD

**Discussion by zhuongxt** [share121/inter-knot#1983] [其他]
wwwww终于祖册神网了wwwwwwwww

**Comment by zhuongxt** [comment]
字打错了

**Comment by Dave-12138** [comment]
> 字打错了

![图片](https://s2.loli.net/2024/08/13/ynSD7l8E6tcbYUI.webp)
右上角三个点，点edit可以修改你发的内容
![流萤 也挺好](https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png)
